bin/make-base-vm

```diff
@@ -193,12 +197,19 @@ if [ $DOCKER = "1" ]; then
     base_image="$DISTRO:$SUITE"
   fi
 
+  apt_cacher=""
+  if [ "$APT_CACHER" = 1 ]; then
+    apt_cacher="RUN echo 'Acquire::http { Proxy "$MIRROR_BASE"; };' > /etc/apt/apt.conf.d/50cacher"
```

Comment:
This line introduces a syntax error when using apt-cacher.

Within the `50cacher` file of the Dockerfile, will render :
```
Acquire::http { Proxy http://172.17.0.1:3142; };
```
instead of
```
Acquire::http { Proxy "http://172.17.0.1:3142"; };
```

Need to escape quotes : `{ Proxy \"$MIRROR_BASE\"; }`.

Fixed in https://github.com/devrandom/gitian-builder/pull/253/commits/525e33acdb15a6c7b2b261d06c2d301318e3d753